lib/puppet/functions/get_rsan_ip.pp

```diff
@@ -0,0 +1,14 @@
+function Functions::get_rsan_ip() {
+  if $settings::storeconfigs {
```

Comment:
$settings::storeconfigs appears to be a reference to code that exists in aa different module, i wouldn't make this conditional on this being true

Reply:
Nah the `$settings::{something}` variable is a global variable added by the compiler: https://puppet.com/docs/puppet/7.1/lang_facts_builtin_variables.html#server-variables